hw/ip/prim/dv/prim_prince/data/cover.cfg

```diff
@@ -0,0 +1,5 @@
++module prim_prince
+begin tgl(portsonly)
+  -tree prim_prince_tb
+  +module prim_prince
```

Comment:
Why only dump toggle coverage?

Reply:
This should dump line/branch/toggle coverage? I just limit toggle coverage to the top level I/o ports.

Reply:
ohh, my bad. I misread it.